tests/Doctrine/Tests/ORM/Persisters/BasicEntityPersisterCompositeTypeSqlTest.php

```diff
@@ -55,4 +56,17 @@ public function testSelectConditionStatementIn()
     {
         $this->_persister->getSelectConditionStatementSQL('admin1', [], [], Comparison::IN);
     }
+
+    /**
+     * this represents the case that occurs when getSelectConditionSQL() is called with Criteria[] as $criteria
+     * as used by findBy([$criteris]) with Critia[] given as argument
```

Comment:
```suggestion
     * as used by findBy([$criterias]) with Criteria[] given as argument
```

Reply:
Isn't "criteria" already in plural form?

Reply:
You're right, fixed! I hope people will not get confused though :sweat_smile: Also, I removed the brackets, is that correct?

Reply:
Seems correct, although I think that this change doesn't make sense because we already have `EntityRepository#matching()` that provides such functionality

Reply:
Oh that's right!